lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1731,6 +1784,8 @@ def main():
         io_threads_enabled=dict(type='bool', default=None),
         ballooning_enabled=dict(type='bool', default=None),
         rng_device=dict(type='str'),
+        numa_tune_mode=dict(type='str', choices=['interleave', 'preferred', 'strict']),
+        numa_node=dict(type='dict', default=[]),
```

Comment:
default=None

Reply:
Shouldn\t this be list, I guess there could be more numa_nodes configurations , no ?

Reply:
Done

Reply:
There will probably be a list of numa nodes but the numanodes service add one node each time (<vm_numa_node>....</vm_numa_node>) so I thought we should act the same here. i.e. update the vm by calling the module each time we want to add another numa node. Is it a problem?

Reply:
Just note that you shouldn't change the type of the parameter in future, because it will break the backward compatibility, so if there we should support list, it should be list now, even if we will add just one value from that list.